### PR TITLE
Store users configuration data in local JSON file instead of MySQL

### PR DESCRIPTION
**Pull request by vladimirkosmala**

### CODE REVIEW

**Comment by awegrzyn** [comment]
Looks good, I will just add travis test and we can merge.

File: QualityControl/test/mocha-config.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      assert.strictEqual(TEST_LAYOUT.owner_id, layout.owner_id);
+      done();
+    });
+  });
+
+  it('Create layout with the same name', (done) => {
+    let layout = TEST_LAYOUT;
+    layout.id = 321;
+    jsonConfig.createLayout(layout).then(() => {
+      done(); // THIS SHOULD NOT BE POSSIBLE !!!
```
Comment (awegrzyn) [inline QualityControl/test/mocha-config.js]: This should not pass

**Comment by awegrzyn** [comment]
I'd also rename `writeLayout` to `updateLayout`

File: QualityControl/test/mocha-config.js (hunk 1 above)
Comment (vladimirkosmala) [reply QualityControl/test/mocha-config.js]: dd

**Comment by awegrzyn** [comment]
Okay, but I think now it's a bit too strict, all users share the same layout name space. Two different users should be allowed setting the same name.

**Comment by vladimirkosmala** [comment]
@awegrzyn ready

**Comment by vladimirkosmala** [comment]
EXDEV error come from the fact that rename() should be atomic but it's impossible if both locations are in two separate disk.

I will place the temp file on same directory.

**Pull request merged**